models/intel/bert-large-uncased-whole-word-masking-squad-0001/description/bert-large-uncased-whole-word-masking-squad-0001.md

```diff
@@ -14,35 +14,33 @@ Tokenization occurs using the BERT tokenizer (see the demo code for implementati
 
 | Metric                    | Value         |
 |---------------------------|---------------|
 | F1                        |        93.21% |
 | Exact match (EM)          |        87.20% |
 
-
 ## Input
 
-Input 0: A `1,384` sequence of tokens (integer values) representing the tokenized premise and question ("input_ids"). The sequence structure is as follows (`[CLS]`, `[SEP]` and `[PAD]` should be replaced by corresponding token IDs as specified by the dictionary):
+1. Input 0: A `1, 384` sequence of tokens (integer values) representing the tokenized premise and question (`input_ids`). The sequence structure is as follows (`[CLS]`, `[SEP]` and `[PAD]` should be replaced by corresponding token IDs as specified by the dictionary):
 `[CLS]` + *tokenized question* + `[SEP]` + *tokenized premise of the question* + `[SEP]` + (`[PAD]` tokens to pad to the maximum sequence length of 384)
 
-Input 1: A `1,384` sequence of integer values representing the mask of valid values in the input ("input_mask"). The values of this input are are equal to:
-1) `1` at positions corresponding to the `[CLS]` + *tokenized question* + `[SEP]` + *tokenized premise of the question* + `[SEP]` part of the Input 0  (i.e. all positions except those containing the `[PAD]` tokens), and
-2) `0` at all other positions
+2. Input 1: A `1, 384` sequence of integer values representing the mask of valid values in the input (`input_mask`). The values of this input are are equal to:
+    * `1` at positions corresponding to the `[CLS]` + *tokenized question* + `[SEP]` + *tokenized premise of the question* + `[SEP]` part of the Input 0  (i.e. all positions except those containing the `[PAD]` tokens), and
+    * `0` at all other positions
 
-Input 2: A `1,384` sequence of integer values representing the segmentation of the Input 0 into question and premise ("segment_ids"). The values are equal to:
-1) `1` at positions corresponding to the *tokenized premise of the question* + `[SEP]` part of the Input 0, and
-2) `0` at all other positions
+3. Input 2: A `1, 384` sequence of integer values representing the segmentation of the Input 0 into question and premise (`segment_ids`). The values are equal to:
+    * `1` at positions corresponding to the *tokenized premise of the question* + `[SEP]` part of the Input 0, and
+    * `0` at all other positions
 
 ## Output
 
-Output 0: The `1, 384` floating point-valued logit scores, where i-th value corresponds to the log-likelihood of the answer to the question starting at the i-th token position of the input.
+1. Output 0: The `1, 384` floating point-valued logit scores, where i-th value corresponds to the log-likelihood of the answer to the question starting at the i-th token position of the input.
 
-Output 1: Same as Output 0, but represents the log-likelihoods of the answer ending at i-th token position.
+2. Output 1: Same as Output 0, but represents the log-likelihoods of the answer ending at i-th token position.
```

Comment:
It should be described separately from `Output 0`, but could be done in new PR.

Reply:
Done